modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/error/ConstructionError.scala

```diff
@@ -11,13 +11,23 @@ sealed trait ConstructionError extends NoStackTrace
 object ConstructionError {
   implicit class ConstructionErrorOps(ce: ConstructionError) {
     def toRosettaError: RosettaError = ce match {
-      case InvalidPublicKey     => RosettaError.InvalidPublicKey
-      case MalformedTransaction => RosettaError.MalformedTransaction
-      case UnsupportedOperation => RosettaError.UnsupportedOperation
+      case IncorrectNumberOfOperations => RosettaError.InvalidRequestBody("Exactly 2 Operations required")
```

Comment:
either change `IncorrectNumberOfOperations` to name specifically about two operations, or change the error msg not to specify the number of operations.

Reply:
@sdavidp resolved with no fix or comment, unresolving